packages/backend/src/models/QueryHistoryModel.ts

```diff
@@ -59,6 +64,19 @@ export class QueryHistoryModel {
         };
     }
 
+    async update(
+        queryUuid: string,
+        projectUuid: string,
+        userUuid: string,
+        queryHistory: Partial<DbQueryHistoryIn>,
```

Comment:
We should have a type for update on `QueryHistoryTable` that is a subset of properties from `DbQueryHistoryIn`.